x/rewards/keeper/rewards.go

```diff
@@ -20,8 +20,8 @@ func (k Keeper) DistributeBlockReward(ctx sdk.Context) {
 	blocksToNextTimerExpiry := k.BlocksToNextTimerExpiry(ctx)
 
 	// get validator distribution pool balance
-	distributionPoolBalance := k.TotalPoolTokens(ctx, types.ValidatorsRewardsDistributionPoolName, k.stakingKeeper.BondDenom(ctx))
-
+	coins := k.TotalPoolTokens(ctx, types.ValidatorsRewardsDistributionPoolName)
+	distributionPoolBalance := coins.AmountOf(k.stakingKeeper.BondDenom(ctx))
```

Comment:
since leftovers and % if the iprpc rewards come o this pool it is wrong to only take the bond denom